cfme/common/vm.py

```diff
@@ -280,8 +280,14 @@ def open_console(self, console='VM Console', invokes_alert=None):
 
         view = navigate_to(self, 'Details')
 
-        # Click console button given by type
-        view.toolbar.access.item_select(console, handle_alert=invokes_alert)
+        # dismiss_any_alerts() call closed subsequent alerts needed for vmrc
+        # below code is needed to fix such issue
+        try:
+            view.browser.IGNORE_SUBSEQUENT_ALERTS = True
```

Comment:
would it be worthwhile to replace this try/finally with appropriate context manager ? :thinking:

Reply:
@izapolsk I think so, it could have uses elsewhere. But I'm okay with that being a separate PR.